### DISCUSSION TITLE
Mention the Firefox version currently being used in the "About Floorp" section

### DISCUSSION THREAD

**Discussion by alok-old** [Floorp-Projects/Floorp#496] [Feature Request]
Just like other Chromium-based browsers where they mention the version of Chromium it's based on, wouldn't it be better to also mention the Firefox version in the "about" section? It would be helpful for users to easily know the version and to look through the fixes and changes it brings directly from the source website. Furthermore, directly adding the link to the source release page of the update will definitely be beneficial.

**Comment by Ryosuke-Asano** [comment]
FIrefox's update link ?

**Comment by alok-old** [comment]
Sorry I meant Release note, just like this [Firefox 115.0esr](https://www.mozilla.org/en-US/firefox/115.0esr/releasenotes/)

**Comment by Stop-Unsecure-ZeroInstall-Updating** [comment]
Dear alok-debnath,

I consider, you and others all over the world using great Floorp, need or hope to know for some purposes, for example, to compare the codes and the functions between Floorp release, and Firefox/Gecko release in the help section?

In addition,  the information on the original codes written by the Floorp development team and the names and licenses of the external software packages merged into Floorp are hopefully to be written in the help section too. 

However, Floorp has its original functions. For example, the vertical tab window controlling function ordinarily just a plugin was developed as Floorp's original function. These codes are not in Firefox/Gocko. So the explanations for these differences are also hopefully in the help.  

Lastly, I need to note that writing Release Notes is not easy, maybe harder than writing codes especially for people always writing codes.
 I don't know the beneficial functions of GitHub, however, finally, these people must check, double-check and lastly write them as simple sentences. 

If you find what is your thought, please point it out without any concerns.

Yours sincerely,
away


But, ***Just I  guess***,  the codes and functions of Floop may be derived from the other versions of Firefox/Gecko.


Anyway, I do not know the real procedure. only reading the codes for this function, not in the help sections on Floop itself. I don't know the real procedure.

**Reply by alok-old** [reply]
Dear @Stop-Unsecure-ZeroInstall-Updating,

I noticed that you have missed almost all the things I said in my previous message. I never mentioned anything about writing the release note, nor does anyone have the desire to compare written codes and changes. What I was trying to convey is that it would be beneficial to include the version of Firefox upon which this is based. This is important because those versions often include critical fixes and significant changes that users may want to refer to. 

Allow me to illustrate this point with an example. Let's suppose a vulnerability has been discovered, but it has already been fixed in a certain version of Firefox. In such a case, it would be invaluable for users to know the specific version their browser is based on. By including this information and providing a link to the release notes of that particular version, users can easily access the relevant information they need. 

Apologies if my previous message was not clear but I don't presume that's the case.

**Comment by Ryosuke-Asano** [comment]
Floorp 11.5.0 show Firefox's version

![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/305cd824-e5d3-4e03-aebd-7d305c0f0d6d)

**Reply by alok-old** [reply]
That's great, thanks
